src/pages/iou/request/MoneyRequestParticipantsSelector.tsx

```diff
@@ -547,7 +550,7 @@ function MoneyRequestParticipantsSelector({
 
     return (
         <>
-            {softPermissionModalVisible && (
+            {!lastLocationPermissionPrompt && softPermissionModalVisible && (
```

Comment:
@perunt with this change, the soft modal will be never show again if we don't clear ONYX storage